app/views/course/condition/achievements/_form.html.slim

```diff
@@ -1,7 +1,11 @@
+- if @conditional.is_a?(Course::Achievement)
```

Comment:
Would it be better if we did the dispatch in new/edit instead? It could also be done based on class name instead of with if/else.

new/edit would then check `@conditional` and render `course/achievement_condition`, which would in turn render the common `form`. This way conditional-specific stuff (right now, only breadcrumbs) would be handled in separate files.